testctrl/cmd/svc/main.go

```diff
@@ -15,68 +15,67 @@
+	"github.com/grpc/grpc/testctrl/kubernetes"
+	"github.com/grpc/grpc/testctrl/svc"
+	lrPb "google.golang.org/genproto/googleapis/longrunning"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
 )
 
 func setupProdEnv(adapter *kubernetes.Adapter) {
-  if err := adapter.ConnectWithinCluster(); err != nil {
-    log.Fatalf("Unable to connect to kubernetes API within cluster: %v", err)
-  }
+	if err := adapter.ConnectWithinCluster(); err != nil {
+		log.Fatalf("Unable to connect to kubernetes API within cluster: %v", err)
+	}
 }
 
 func setupDevEnv(adapter *kubernetes.Adapter, grpcServer *grpc.Server) {
-  c, set := os.LookupEnv("KUBE_CONFIG_FILE")
-  if !set {
-    log.Fatalf("Missing a kube config file, specify its absolute path in the KUBE_CONFIG_FILE env variable.")
-  }
-  if err := adapter.ConnectWithConfig(c); err != nil {
-    log.Fatalf("Invalid config file specified by the KUBE_CONFIG_FILE env variable, unable to connect: %v", err)
-  }
+	c, set := os.LookupEnv("KUBE_CONFIG_FILE")
+	if !set {
+		log.Fatalf("Missing a kube config file, specify its absolute path in the KUBE_CONFIG_FILE env variable.")
+	}
+	if err := adapter.ConnectWithConfig(c); err != nil {
+		log.Fatalf("Invalid config file specified by the KUBE_CONFIG_FILE env variable, unable to connect: %v", err)
+	}
 
-  log.Println("Enabling reflection for grpc_cli; avoid this flag in production.")
-  reflection.Register(grpcServer)
+	log.Println("Enabling reflection for grpc_cli; avoid this flag in production.")
+	reflection.Register(grpcServer)
 }
 
 func main() {
-  port := flag.Int("port", 50051, "Port to start the service.")
-  flag.Parse()
+	port := flag.Int("port", 50051, "Port to start the service.")
+	flag.Parse()
 
-  grpcServer := grpc.NewServer()
-  adapter := &kubernetes.Adapter{}
+	grpcServer := grpc.NewServer()
+	adapter := &kubernetes.Adapter{}
 
-  env := os.Getenv("APP_ENV")
-  if strings.Compare(env, "production") == 0 {
-    log.Println("App environment set to production")
-    setupProdEnv(adapter)
-  } else {
-    log.Println("App environment set to development")
-    setupDevEnv(adapter, grpcServer)
-  }
+	env := os.Getenv("APP_ENV")
+	if strings.Compare(env, "production") == 0 {
+		log.Println("App environment set to production")
+		setupProdEnv(adapter)
+	} else {
+		log.Println("App environment set to development")
+		setupDevEnv(adapter, grpcServer)
+	}
 
-  lis, err := net.Listen("tcp", fmt.Sprintf(":%d", *port))
-  if err != nil {
-    log.Fatalf("Failed to listen on port %d: %v", *port, err)
-  }
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", *port))
+	if err != nil {
+		log.Fatalf("Failed to listen on port %d: %v", *port, err)
+	}
 
-  lrPb.RegisterOperationsServer(grpcServer, &svc.OperationsServer{})
-  pb.RegisterTestSessionsServer(grpcServer, &svc.TestSessionsServer{adapter})
+	lrPb.RegisterOperationsServer(grpcServer, &svc.OperationsServer{})
+	pb.RegisterSchedulingServiceServer(grpcServer, &svc.SchedulingServiceServer{})
```

Comment:
I don't think it is necessary to include "Service" here, SchedulingServer and SchedulingClient may be enough.

Reply:
Agreed. This is why I'm not a huge fan of the service suffix. It generates code like this (line 622 in generated):

```
type UnimplementedSchedulingServiceServer struct {}
```